tools/toktx/toktx.cc

```diff
@@ -1257,6 +1266,36 @@ toktxApp::main(int argc, _TCHAR *argv[])
                 exitCode = 2;
                 goto cleanup;
             }
+        } else if (options.astc) {
+            commandOptions::astcOptions& astcopts = options.astcopts;
+#if TRAVIS_DEBUG
+            astcopts.print();
+#endif
+            if (options.inputSwizzle.size()) {
+                for (i = 0; i < 4; i++) {
+                     astcopts.inputSwizzle[i] = options.inputSwizzle[i];
+                }
+            } else if (defaultSwizzle.size()) {
+                 for (i = 0; i < 4; i++) {
+                     astcopts.inputSwizzle[i] = defaultSwizzle[i];
+                }
+            }
+
+            if (firstImageOETF == KHR_DF_TRANSFER_SRGB) {
```

Comment:
You should be using chosenOETF here.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/433/commits/18af4983456764eb784babe2d11a125f5d2b7f5c